test/test_spectral_ops.py

```diff
@@ -506,6 +506,78 @@ def test_fftn_invalid(self, device):
         with self.assertRaisesRegex(RuntimeError, "Expected a real input"):
             torch.fft.rfftn(c)
 
+    # Helper functions
+
+    @skipCPUIfNoMkl
+    @skipCUDAIfRocm
```

Comment:
XLA build is currently failing for unrelated reasons but you may need to skip this test on XLA (by testing it only on CPU and CUDA backends).

Reply:
(same for the following tests)